src/templates/next/components/shared/Notification/Notification.tsx

```diff
@@ -0,0 +1,30 @@
+import { NotificationProps } from "~/common/Notification"
```

Comment:
nit: can export the props in `~/common/index.ts` so the import here is simplified

Reply:
[88000ab](https://github.com/isomerpages/isomer-components/pull/67/commits/88000ab71881e74a26f77b82c12b38498b6edca1)